tests/test_protocols.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Oct 21 12:16:49 2022
+
+@author: Alexandra N. Walker
+
+Testing methods from protocols.py
+"""
+from controller.protocols import (
+    connection,
+    didexchange,
+    indy_anoncred_onboard,
+    indy_anoncred_credential_artifacts,
+    indy_issue_credential_v1,
+    indy_issue_credential_v2,
+    indy_present_proof_v1,
+    indy_present_proof_v2,
+    indy_anoncreds_revoke,
+    indy_anoncreds_publish_revocation,
+    indy_anoncreds_proof_revocations
+    )
+from tests.conftest import getenv_or_raise
+from controller.controller import Controller
+import pytest
+import pytest_asyncio
+from os import getenv
+
+@pytest_asyncio.fixture
+async def alice():
+    alice = await Controller(getenv_or_raise("ALICE")).setup()
+    yield alice
+    await alice.shutdown()
+
+@pytest_asyncio.fixture
+async def bob():
+    bob = await Controller(getenv_or_raise("BOB")).setup()
+    yield bob
+    await bob.shutdown()
```

Comment:
These fixtures are already defined in `conftest.py`; it's not strictly hurting anything for these to be here but if we needed to change something about the fixture, we'd 'have to change it in both locations. We can eliminate these fixtures and use the global ones from conftest.py.